debbuild.spec

```diff
@@ -53,29 +51,10 @@ rebuild .src.rpm source packages as .deb binary packages.
 # Transfer $VERSION into the live system
 %{__perl} -pe "s/\@VERSION\@/%{version}/" -i debbuild
 
-%install
-# Steps to install to a temporary location for packaging
-%{__rm} -rf %{buildroot}
-
-%{__install} -d %{buildroot}%{_bindir}
-%{__install} debbuild %{buildroot}%{_bindir}
-
-%{__install} -d %{buildroot}%{_libdir}/%{name}
-%{__install} -m 644 macros/macros.in %{buildroot}%{_libdir}/%{name}/macros
-%{__install} -m 644 config/debrc %{buildroot}%{_libdir}/%{name}
+%configure NAME=%{name} VERSION=%{version}
```

Comment:
It felt wrong to pass `--name` and especially `--version` as configure options, so I'm implementing them as variables, but this seems potentially nonstandard however we slice it. Suggestions welcome.